acnportal/acnsim/simulator.py

```diff
@@ -86,6 +90,11 @@ def __init__(
             self.max_recompute = scheduler.max_recompute
             self.scheduler.register_interface(interface_type(self))
 
+    def generate_interface(self, interface_type: type) -> Interface:
```

Comment:
@zach401 For cases when an input scheduler is not given to the Simulator, we still need a way to get an interface to a simulator (for, e.g. RL training in which a simulation is stepped). If this looks good I'll write a test

Reply:
What do you think about `get_interface()` instead?
I also don't understand the doc string, why does it matter if scheduler is None? This function doesn't case what the scheduler is, right?

Reply:
I had anticipated that it would only ever be called when `scheduler` is `None` because otherwise, the `Interface` is accessible through `scheduler.interface`. That being said, the method itself doesn't require that, so I'll update the docstring and change to `get_interface`.